test/integration/connect/envoy/consul-base-cfg/base.hcl

```diff
@@ -1 +1,2 @@
-primary_datacenter = "primary"
\ No newline at end of file
+primary_datacenter = "primary"
+log_level          = "trace"
```

Comment:
If we keep the `logTrace*` functions, this is super helpful because it'll log the full protocol chatter to TRACE.